src/components/Asset/AssetActions/TermsAndConditionsCheckbox.tsx

```diff
@@ -4,31 +4,34 @@ import styles from './index.module.css'
 
 export default function TermsAndConditionsCheckbox({
   name,
-  actions,
+  license,
   onChange,
   disabled,
   options,
   prefixes,
   postfixes
 }: {
   name: string
-  actions?: any
+  license: string[]
```

Comment:
suggestion(non-blocking): renaming to `licenses` (plural) communicates a bit better that this allows array values